### ISSUE TITLE
Add option for explicit input file

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgosm-flex#192]
# Details

Allow the use of an explicit PBF input file as an option instead of requiring the Geofabrik naming scheme and associated `.md5` file. This should technically allow any format of OSM input file that osm2pgsql supports.

## History

@jacopofar submitted a working proof of concept of this in #186 along with the option of using an externally defined Postgres connection instead of the in-Docker Postgres connection.  The portion of #186 associated with this issue was relatively straightforward.  The option was added in #186 as `--input-file`.

```python
@click.option('--input-file',
              required=False,
              default=None,
              help='Path of the input PBF file')
```

There are only a couple places that need logic adding when `--input-file` is set.

> Note: It was the connection portion of that PR (#186) that turned out to be complicated as a side-effect of the overall messy structure in `docker/pgosm_flex.py`.  The structure there is largely the way it was when it was rewritten from its original bash script to Python.  These other details from #186 are under #193 and #194.

## Other notes

* [x] Error handling - concise message when user inputs path that doesn't exist
* [x] Update `docs/DOCKER_RUN.md` with example usage
* [x] Example from `--help` should show `--input-file`
* [x] Original version in #186 included logic RE log file, that is no longer needed after #189

**Comment by rustprooflabs** [comment]
The main part of this is in the `main` branch, the `latest` Docker image has been refreshed.  I don't believe there is any error handling RE a nonexistent `--input-file`, I'll come back to that part soon.
